functions/src/index.ts

```diff
@@ -131,3 +131,20 @@ exports.initRepoPRs = firestore.document('repositories/{id}').onCreate((snapshot
     console.error(err);
   });
 });
+
+exports.deleteCachedConfigs = https.onRequest(async (request: Request, response: Response) => {
```

Comment:
Why is this fn here? It doesn't seem to be used by anything. It's also missing docs. Can you please explain and document? Thanks

Reply:
This will delete the config cached in firebase. It's just in case, if we want to force the bot to do a hard refresh of the config from github (for example if the config changed, but the push event wasn't delivered, then the config in firebase is wrong and we need to force a refresh).
I can clean the firebase database manually, but with this function anyone without access to firebase can also force the refresh (could be useful for caretakers?).
I'll add a comment.